src/modules/include/gradient.fh

```diff
@@ -1061,7 +1061,7 @@ contains
              weight=quick_dft_grid%gridb_weight(Igp)
              Iatm=quick_dft_grid%gridb_atm(Igp)
               
-              if (weight < quick_method%XCCutoff ) then
+              if (weight < quick_method%DMCutoff ) then
```

Comment:
I think we should add another variable XCWeightCutoff.

Reply:
Scratch that. We are just screening for zeros to matrix elements or gradient contributions. Although at some point we might want to revisit naming of DMCutoff.